outputs.tf

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+output "accounts" {
+  description = "Map of account names to account IDs"
+  value = merge(
+    { "payer" = aws_organizations_account.payer.id, "security" = module.security_account.account_id },
```

Comment:
```suggestion
    {var.organization["payer_name"] = aws_organizations_account.payer.id, var.organization["security_account_name"] = module.security_account.account_id },
```

Reply:
Since we have the user's supplied names, should we do this instead?

Reply:
Good point.